diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -1431,3 +1436,20 @@ void DifferentialFunctionComparator::findDifference(
         }
     }
 }
+
+bool DifferentialFunctionComparator::equal(const Instruction *InstL,
+                                           const Instruction *InstR) {
+    auto L = sn_mapL.find(InstL);
+    auto R = sn_mapR.find(InstR);
+
+    if (L == sn_mapL.end() || R == sn_mapR.end())
+        // Values are not in maps
+        return false;
+
+    if (std::make_pair(InstL, InstR) == DifferingInstructions)
```

Comment:
```suggestion
    if (DifferingInstructions == {InstL, InstR})
```

Reply:
Not working.

Reply:
Yes, I believe that this might be invalid. I don't think you can compare directly against an `std::initializer_list`.

Reply:
clang for example explicitly reports this:
```
$ clang test.cpp -o test
test.cpp:5:16: error: initializer list cannot be used on the right hand side of operator '=='
  return (test == {1, 2});
               ^  ~~~~~~
1 error generated.
```